sklearn/covariance/outlier_detection.py

```diff
@@ -69,7 +70,7 @@ def decision_function(self, X, raw_values=None):
         decision : array-like, shape (n_samples, )
             The values of the decision function for each observations.
             It is equal to the Mahalanobis distances if `raw_values`
-            is True. By default (``raw_values=True``), it is equal
+            is True. By default (``raw_values=False``), it is equal
```

Comment:
I'm assuming this was a typo, since `raw_values=False` is the default, and the behavior described is for `raw_values=False`...